curated_data/payer_map.csv

```diff
@@ -303,3 +303,65 @@ GENERIC WORKERS COMP,Worker's Comp,95,Worker's Compensation
 MS WORKERS COMP,Worker's Comp,95,Worker's Compensation
 WORKERS COMP,Worker's Comp,95,Worker's Compensation
 WORKMAN'S COMPENSATION,Worker's Comp,95,Worker's Compensation
+AIH COLLECTIONS,,349,Other
```

Comment:
Payers without a financial class were added for IDX.

O2 joins on both payer name and financial class, while IDX only joins on payer name.  There are comments in the encounter code about this asymmetric join.  

Strictly speaking, financial class isn't needed, as long as one payer name can't map to more than one CDM code, but I kept both in the map as financial class ultimately has more influence on the choice of code.